.github/workflows/template.yml

```diff
@@ -0,0 +1,22 @@
+name: Merge template
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  workflow_dispatch:
+
+jobs:
+  merge_template:
+    runs-on: ubuntu-latest
+    container: obolibrary/odkfull:v1.2.26
+    steps:
+      - uses: actions/checkout@v2
+
```

Comment:
```suggestion
        with:
          persist-credentials: false # otherwise, the token used is the GITHUB_TOKEN, instead of your personal token
          fetch-depth: 0 # otherwise, you will failed to push refs to dest repo

````